AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1317,6 +1317,7 @@ slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,itprotoday.com
 bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.com,swindonadvertiser.co.uk,wiltshiretimes.co.uk##.cta-component-full-width
 !
 camstrans.ru###right-column > section.right-box
+geo.de##.nlAnmeldg
```

Comment:
I don't see it on the page
@BlazDT please take a look

Reply:
Same

Reply:
<details>
<summary>Newsletter block</summary>
<img src="https://user-images.githubusercontent.com/18536328/75993710-f56b8f00-5f0a-11ea-87d1-9ca27be12da8.png" />
</details>

Reply:
OK